.github/pr-template.md

```
@@ -0,0 +1,169 @@
+  body: JSON.stringify({
+    prompt: "a beautiful sunset",
+    model: "gpt-image-2",
+    upscale: "2k"  // "" / "2k" / "4k"
+  })
+});
+```
+
+## 📊 性能指标
+
+### 图片放大性能
+| 配置 | 原图生成 | 2K 放大 | 4K 放大 |
```

markdownlint-cli2 (0.22.1)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)